monolith/services/restaurant_services.py

```diff
@@ -111,3 +111,23 @@ def get_reservation_rest(owner_id, restaurant_id, from_date, to_date, email):
         # execute and retrive results...
         result = db.engine.execute(stmt, params)
         return result.fetchall()
+
+    @staticmethod
+    def reviewRestaurant(restaurant_id, reviewer_id, stars, review):
```

Comment:
you can use the convention with review_estaurant?

Reply:
ok modifico